src/server/routes/auth.ts

```diff
@@ -20,23 +53,23 @@ if (Environment.getFacebookAuth()) {
 if (Environment.getGithubAuth()) {
     router.get('/github/login', passport.authenticate('github', { scope: ['user:email'] }));
 
-    router.get('/github/callback', passport.authenticate('facebook', {
+    router.get('/github/callback', passport.authenticate('github', {
         failureRedirect, successRedirect,
     }));
 }
 
 if (Environment.getGoogleAuth()) {
     router.get('/google/login', passport.authenticate('google', { scope: ['email', 'profile'] }));
 
-    router.get('/google/callback', passport.authenticate('facebook', {
+    router.get('/google/callback', passport.authenticate('google', {
         failureRedirect, successRedirect,
     }));
 }
 
 if (Environment.allowLocalAuth()) {
-    router.post('/login', passport.authenticate('local', { failureRedirect: '/login', successRedirect: '/' }));
+    router.post('/login', postParser, passport.authenticate('local', { failureRedirect: '/login', successRedirect: '/' }));
```

Comment:
we can keep this for now but we should look into how to do this without reloading the page.

Reply:
#22